esp-hal/src/otg_fs.rs

```diff
@@ -49,16 +47,20 @@ use crate::{
 };
 
 #[doc(hidden)]
+/// Trait representing the USB D+ (data plus) pin.
```

Comment:
Hold on, these trigger the warning even with `#[doc(hidden)]`? 🤯

Reply:
Nope, just wanted to leave a comment there as well.